src/python/pants/backend/project_info/tasks/export_dep_as_jar.py

```diff
@@ -30,6 +30,9 @@ class ExportDepAsJar(ConsoleTask):
 
   This is an experimental task that mimics export but uses the jars for
   jvm dependencies instead of sources.
+
+  This goal affects the contents of the runtime_classpath, and should not be
+  combined with any other goals on the command line.
```

Comment:
I think this comment could go better in whatever `register.py` file registers this task.

Reply:
👍